src/js/components/about/NextStepsBox.jsx

```diff
@@ -5,6 +5,7 @@
 
 import React from 'react';
 import * as AboutIcons from 'components/sharedComponents/icons/about/AboutIcons';
+import * as Icons from 'components/sharedComponents/icons/Icons';
```

Comment:
Please remove this if we aren't using it anywhere. This is an unused import, which is failing the linter.